promote_image.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+#This scripts is used to push the images to repository
+
+IMAGE_NAME=$1
+IMAGE_PROD=$2
+IMAGE_LATEST=$3
+
+docker pull ${IMAGE}
```

Comment:
You're not setting the `${IMAGE}` variable. I think you mean `${IMAGE_NAME}`

Reply:
Likewise, I'd want more robustness around the arguments here including checking for valid choices and ensuring it errors out with an appropriate message if needed.

Reply:
@bencourliss you are right, it should be `${IMAGE}`, Thanks